modules/core/src/views/first-person-view.js

```diff
@@ -25,7 +25,7 @@ export default class FirstPersonView extends View {
       bearing = 0,
       pitch = 0,
       up = [0, 0, 1] // Defines up direction, default positive z axis,
-    } = props.viewState;
+    } = props;
```

Comment:
Nit: Are the props now in fact a view state? Change the name of the param? 

Can't tell as I am reading this, so not sure, just a suggestion...